docs/source/partials/examples/_address_search.html.erb

```diff
@@ -1,8 +1,4 @@
-<input
-  type="text"
-  id="address"
-  placeholder="Where are we going?"
-/>
```

Comment:
Eheh this was done only to avoid horizontal scrollbars on code examples, code examples are very narrow unfortunately in term of code samples